java/piranha/src/test/java/com/uber/piranha/XPFlagCleanerTest.java

```diff
@@ -2120,4 +2120,136 @@ public void exceptionWhenArgumentIndexNotSpecifiedAndNotArgumentIndexOptional()
     XPFlagCleaner flagCleaner = new XPFlagCleaner();
     flagCleaner.init(b.build());
   }
+
+  @Test
+  public void testEmptyFlagRemovesStaticImports() throws IOException {
+
+    ErrorProneFlags.Builder b = ErrorProneFlags.builder();
+    b.putFlag("Piranha:FlagName", "");
+    b.putFlag("Piranha:IsTreated", "true");
+    b.putFlag("Piranha:Config", "config/properties.json");
+
+    BugCheckerRefactoringTestHelper bcr =
+        BugCheckerRefactoringTestHelper.newInstance(new XPFlagCleaner(b.build()), getClass());
+
+    bcr = bcr.setArgs("-d", temporaryFolder.getRoot().getAbsolutePath());
+
+    bcr = addHelperClasses(bcr);
+    bcr.addInputLines(
+            "EmptyFlagRemovesStaticImports.java",
+            "package com.uber.piranha;",
+            "import static com.uber.piranha.Constants.ONE;",
+            "class EmptyFlagRemovesStaticImports {",
+            "  public String evaluate(int x) {",
+            "    if (x == ONE) { return \"yes\"; }",
+            "    return \"no\";",
+            "  }",
+            "}")
+        .addOutputLines(
+            "EmptyFlagRemovesStaticImports.java",
+            "package com.uber.piranha;",
+            "import static com.uber.piranha.Constants.ONE;",
+            "class EmptyFlagRemovesStaticImports {",
+            "  public String evaluate(int x) {",
+            "    if (x == ONE) { return \"yes\"; }",
+            "    return \"no\";",
+            "  }",
+            "}")
+        .addInputLines(
+            "Constants.java",
+            "package com.uber.piranha;",
+            "class Constants {",
+            "  public static int ONE = 1;",
+            "}")
+        .addOutputLines(
+            "Constants.java",
+            "package com.uber.piranha;",
+            "class Constants {",
+            "  public static int ONE = 1;",
+            "}")
+        .doTest();
+  }
+
+  @Test
+  public void testEmptyFlagRemovesAnnotatedMethods() throws IOException {
```

Comment:
nit: test needs to be either renamed to `testEmptyFlagOnAnnotatedMethods` or `testEmptyFlagDoesNotRemoveAnnotatedMethods`. The latter is descriptive but more verbose. If you go for the former, please add a comment before the test.

Reply:
I'll add comments with the latter name.